gamemodes/terrortown/gamemode/client/cl_help/populate_appearance.lua

```diff
@@ -0,0 +1,554 @@
+	bindingsData:SetIcon(materialIcon)
+end
+
+---
+-- Function to populate sub menu
+-- @param table helpData The sub menu data object
+-- @param string id The unique id of this menu element
+-- @internal
+-- @realm client
+HELPSCRN.subPopulate["ttt2_appearance"] = function(helpData, id)
+	-- general
+	local generalData = helpData:PopulateSubMenu(id .. "_general")
+
+	generalData:SetTitle("submenu_appearance_general_title")
+	generalData:PopulatePanel(PopulateGeneralPanel)
+
+	-- HUD switcher
+	local hudData = helpData:PopulateSubMenu(id .. "_hud_switcher")
+
+	hudData:SetTitle("submenu_appearance_hudswitcher_title")
+	hudData:PopulatePanel(PopulateHUDSwitcherPanel)
+	hudData:PopulateButtonPanel(function(parent)
+		local buttonReset = vgui.Create("DButtonTTT2", parent)
+		buttonReset:SetText("Reset")
+		buttonReset:SetSize(100, 45)
+		buttonReset:SetPos(475, 20)
+		buttonReset.DoClick = function(btn)
+			local currentHUD = huds.GetStored(HUDManager.GetHUD())
+
+			if not currentHUD then return end
+
+			currentHUD:Reset()
+			currentHUD:SaveData()
+		end
+
+		local buttonEditor = vgui.Create("DButtonTTT2", parent)
+		buttonEditor:SetText("HUD Editor")
+		buttonEditor:SetSize(175, 45)
+		buttonEditor:SetPos(600, 20)
+		buttonEditor.DoClick = function(btn)
+			local currentHUDName = HUDManager.GetHUD()
+
+			if not currentHUDName then return end
+
+			HUDEditor.EditHUD(currentHUDName)
+
+			VHDL.HideFrame()
+		end
+	end)
+
+	-- VSKIN
+	local vskinData = helpData:PopulateSubMenu(id .. "_vskin")
+
+	vskinData:SetTitle("submenu_appearance_vskin_title")
+	vskinData:PopulatePanel(PopulateVSkinPanel)
+
+	-- targetID
+	local targetData = helpData:PopulateSubMenu(id .. "_target_id")
+
+	targetData:SetTitle("submenu_appearance_targetid_title")
+	targetData:PopulatePanel(PopulateTargetIDPanel)
+
+	local shopData = helpData:PopulateSubMenu(id .. "_shop")
+
+	shopData:SetTitle("submenu_appearance_shop_title")
+	shopData:PopulatePanel(PopulateShopPanel)
+
+	-- crosshair
+	local crosshairData = helpData:PopulateSubMenu(id .. "_crosshair")
+
+	crosshairData:SetTitle("submenu_appearance_crosshair_title")
+	crosshairData:PopulatePanel(PopulateCrosshairPanel)
+
+	-- damage indicator
+	local damageData = helpData:PopulateSubMenu(id .. "_damage_indicator")
+
+	damageData:SetTitle("submenu_appearance_dmgindicator_title")
+	damageData:PopulatePanel(PopulateDamagePanel)
+
+	-- performance
+	local performanceData = helpData:PopulateSubMenu(id .. "_performance")
+
+	performanceData:SetTitle("submenu_appearance_performance_title")
+	performanceData:PopulatePanel(PopulatePerformancePanel)
+
+	-- interface
+	local interfaceData = helpData:PopulateSubMenu(id .. "_interface")
+
+	interfaceData:SetTitle("submenu_appearance_interface_title")
+	interfaceData:PopulatePanel(PopulateInterfacePanel)
+
+	-- miscellaneous
+	local miscellaneousData = helpData:PopulateSubMenu(id .. "_miscellaneous")
+
+	miscellaneousData:SetTitle("submenu_appearance_miscellaneous_title")
```

Comment:
What is the purpose of the miscellaneous sub menu?

Reply:
For now, nothing. If no settings are added until it is released, I will remove it